### PR TITLE
Use Bazelisk to select and download Bazel version

### PR DESCRIPTION
**Pull request by realityforge**

### CODE REVIEW

**Comment by jDramaix** [comment]
I'll check if we can use that for our internal CI as well. It's better to be aligned.

File: .travis.yml [hunk 1]
```diff
@@ -3,8 +3,8 @@ dist: trusty
 sudo: required
 # xcode8 has jdk8
 osx_image: xcode8
-# Not technically required but suppresses 'Ruby' in Job status message.
-language: java
+# Since Bazel 0.25, PY3 is the default Python version. Make sure Travis has it.
```
Comment (niloc132) [inline .travis.yml]: Possibly a stupid question, but if j2cl only can build on bazel 24, does this matter? 

https://github.com/google/j2cl/blob/master/.bazelversion

Comment (jDramaix) [reply .travis.yml]: You can already build j2cl with the last bazel version if you use the flag --incompatible_disable_deprecated_attr_params=false This is due to the dependency to protocol buffer.
They fix the problem https://github.com/protocolbuffers/protobuf/pull/6301 on their side and when we upgrade the version of our dependency, we may change the bazel version to the latest

Comment (niloc132) [reply .travis.yml]: Right, not objecting necessarily to this, just pointing out that this line doesn't seem to make sense, as a few lines later we reference the contents of .bazelversion to download 24.0.

Using .bazelversion is consistently working for us in circleci as well.

Comment (realityforge) [reply .travis.yml]: Hopefully, #67 is applied soon which will update to a later version of Bazel

Comment (jDramaix) [reply .travis.yml]: With https://github.com/google/j2cl/commit/7114599771f8a3a8fe146e3704d425be1a9ee6a4 you don't the specific blaze flag

Comment (realityforge) [reply .travis.yml]: :+1:

**Comment by realityforge** [comment]
@gkdn Is it useful for me to revive this patch now that bazelisk is documented in 9a28d10bb9cf906c5a2dc5a2e19e074eeab198d8 or do you still want to tackle your internal CI first?

**Comment by gkdn** [comment]
It is funny; I was preparing it when you sent the msg. The patch will be here soon.

**Comment by realityforge** [comment]
Woohoo!

**Comment by realityforge** [comment]
And thanks!

**Comment by gkdn** [comment]
Seems like travis is failing for OSX. Do you want to take a look?

**Comment by realityforge** [comment]
I am away this week but will 👀 next week

**Pull request closed without merging**